docs/jetstream/operations.md

```diff
@@ -0,0 +1,129 @@
+---
+id: operations
+title: Jetstream Architecture and Operations
+---
+
+**[Jetstream]** is part of the Cirrus ecosystem and depends on some external services.
+
+<img src="/experimenter-docs/img/jetstream/cirrus.png" alt="Cirrus architecture" className="img-xl" />
```

Comment:
I don't think this is the right relative url for experimenter.info, though maybe Jody will come up with something in https://github.com/mozilla/experimenter/issues/5112.

Reply:
Having a preview in Netlify would come in handy here :D